lib/collection/src/config.rs

```diff
@@ -212,14 +234,36 @@ impl CollectionParams {
             })
     }
 
+    pub fn get_sparse_vector_params_mut(
+        &mut self,
+        vector_name: &str,
```

Comment:
I think we need a different implementation here because sparse vectors do not support anonymous usage

Reply:
Got it, fixed